resources/js/Components/MismatchRow.vue

```diff
@@ -193,18 +198,22 @@ function showDialog(e: Event) {
 <style lang="scss">
 @import "@wikimedia/codex-design-tokens/theme-wikimedia-ui";
 
-  a {
-    break-line-link {
-      width: 100%
-    }
-  }
+a {
+break-line-link {
```

Comment:
Should we use class selector as `.break-line-link`?

Reply:
yes, this is a mistake, i did this accidentally, thanks for the heads up

Reply:
This issue was introduced in this PR: https://github.com/wmde/wikidata-mismatch-finder/pull/772/files#diff-3811cb136ae5a9237926413dcf5ae788c91abfeff0b3eda02d4f2c3fc6b724b3R201

It is good that we caught it, since it hasn't been too long. :+1: